lib/two_bucket_cases.rb

```diff
@@ -8,15 +8,15 @@ def do
     "TwoBucket.new(#{bucket_one}, #{bucket_two}, #{goal}, '#{start_bucket}')"
   end
 
-  def test_body
+  def work_load
     "two_bucket = #{self.do}
     assert_equal #{expected['moves']}, two_bucket.moves
     assert_equal '#{expected['goal_bucket']}', two_bucket.goal_bucket
     assert_equal #{expected['other_bucket']}, two_bucket.other_bucket"
```

Comment:
I'd like to see work_load **not** have to be responsible for whitespace/indenting.